packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -337,6 +338,42 @@ export default class TinyMCE extends Component {
 		}
 	}
 
+	/**
+	 * Replaces non breaking spaces inserted by TinyMCE on the space key by
+	 * a normal space.
+	 *
+	 * @param  {SyntheticEvent} event Synthetic key up event.
+	 */
+	onKeyUp( event ) {
+		if ( event.shiftKey || event.keyCode !== SPACE ) {
+			return;
+		}
+
+		const selection = getSelection();
+
+		if ( selection.rangeCount < 0 ) {
+			return;
+		}
+
+		const range = selection.getRangeAt( 0 );
+		const { startContainer, startOffset, collapsed } = range;
+
+		if ( ! collapsed ) {
+			return;
+		}
+
+		const characterOffset = startOffset - 1;
+		const character = startContainer.data[ characterOffset ];
```

Comment:
Was hard to track down the interface where `data` comes from:

https://dom.spec.whatwg.org/#characterdata
https://developer.mozilla.org/en-US/docs/Web/API/CharacterData

From above, this wouldn't exist on an element node. Is it possible that `startContainer` might be an element?

Reply:
Yes, we should return early then.